### PR TITLE
Expand documentation, Python 3 support

### PR DESCRIPTION
**Pull request by RoelN**
I added some documentation as I couldn't get it up and running without these steps. Also quickly moved everything to Python 3.

### CODE REVIEW

File: pdf2slides.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+if not os.path.isfile("token.json"):
+    drive = GoogleDrive("token.json", "credentials.json", SCOPES)
+    slides = GoogleSlides("token.json", "credentials.json", SCOPES)
+    sys.exit()
+
+if len(sys.argv) < 2:
+    print("Syntax: " + sys.argv[0] + " <pdf file>")
+    sys.exit()
+
+pages = Image(filename=sys.argv[1], resolution=300)
```
Comment (davelab6) [inline pdf2slides.py]: Suggest to make resolution `argv[2]`

File: pdf2slides.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+presentation = slides.createPresentation(sys.argv[1])
+pid = presentation["presentationId"]
+print("Created presentation " + pid)
+
+# Remove the blank first slide
+slides.clearSlides(presentation)
+print("Removed blank slide")
+
+# Populate the slides
+i = 0
```
Comment (davelab6) [inline pdf2slides.py]: Maybe should start from 1?

**Review by davelab6 — APPROVED** [review]

File: pdf2slides.py (hunk 2 above)
Comment (RoelN) [reply pdf2slides.py]: It's only for output during build, so doesn't affect created Slides document, but changed anyway :-)

**Comment by RoelN** [comment]
@elliotkendall I hope you appreciate these tweaks, if you have any questions, please let me know :-)

**Comment by elliotkendall** [comment]
Thanks, folks! I'm glad you found this code useful

**Pull request merged**